src/index.html

```diff
@@ -1,13 +1,311 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Bakery</title>
     <link rel="stylesheet" href="./styles/main.scss">
-  </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap" rel="stylesheet">
     <script type="text/javascript" src="scripts/main.js"></script>
+  </head>
+  <body class="page__body">
+    <header class="header">
+      <div class="top-bar">
+        <a href="#" class="top-bar__logo-link">
+          <img
+            src="images/Logo.png"
+            alt="Logo bakery"
+            class="top-bar__logo"
+          >
+        </a>
+        <a href="#menu" class="icon icon--menu"></a>
+
+        <div class="nav__container">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#products">Products</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#about">About Us</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#blog">Blog</a>
+              </li>
+              <li class="nav__item">
+                <button class="nav__button">
+                  <span class="icon--phone"></span>
+                  <a class="nav__link" href="tel:+16295550129" >
+                    <span>Contact Us</span>
+                  </a>
+                </button>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+
+      <div class="header__container">
+        <div class="header__text">
+          We bake your fantasies — croissants, cakes, cookies, and cupcakes.
+        </div>
+        <div class="header__box">
+          <h1 class="header__title">Creative bakery</h1>
+          <img
+            src="images/Cookie.png"
+            alt="Cookie photo"
+            class="header__image"
+          >
+        </div>
+        <button class="header__button">
+          <span class="icon__cookie"></span>
+          <span>Learn More</span>
+        </button>
+      </div>
+    </header>
+
+    <aside class="page page__menu" id="menu">
+      <div class="top-bar">
+        <a href="#" class="top-bar__logo-link">
+          <img
+          src="images/Logo.png"
+          alt="Logo bakery"
+          class="top-bar__logo"
+        >
```

Comment:
Need to fix and check in everywhere
```suggestion
          <img
            src="images/Logo.png"
            alt="Logo bakery"
            class="top-bar__logo"
          >
```